examples/sfinc_update_model.ipynb

```diff
@@ -0,0 +1,375 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Update existing Sfincs model with boundary conditions and forcing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demonstrates how to update an existing SFINCS model with gauges, boundary conditions and forcing such as precipitation. The notebook does not contain the SFINCS code or executables to run the model with. \n",
+    "\n",
+    "We will perform the following:\n",
+    "* append an existing schematization that only contains static schematization layers\n",
+    "* introduce gridded precipitation only \n",
+    "* introduce gridded precipitation, water level boundaries and observation gauge point locations\n",
+    "* write the new models to new model folders\n",
+    "\n",
+    "This notebook demonstrates the update functionalities on a test model delivered with the hydromt_sfincs plugin."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### HydroMT plugin availability\n",
+    "First let us check if HydroMT has the Sfincs plugin available. The result of the code block below should be\n",
+    "`hydroMT model plugins: sfincs (vx.x.x)` where `x.x.x` denotes the current version (e.g. `0.1.0`). If Sfincs seems missing, ensure it is installed by typing `pip install hydromt_sfincs` on a command line."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!hydromt --models"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### contents of HydroMT Sfincs configuration .ini file\n",
```

Comment:
I moved this part to the other notebook as I think it's nice to show how to build an entire model in that notebook.